pkg/scrape/scrape.go

```diff
@@ -0,0 +1,193 @@
+package scrape
+
+import (
+	"fmt"
+	"github.com/marcus-crane/khinsider/v2/pkg/util"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/pterm/pterm"
+
+	"github.com/marcus-crane/khinsider/v2/pkg/types"
+)
+
+const (
+	LetterBase = "https://downloads.khinsider.com/game-soundtracks/browse/"
+)
+
+func DownloadPage(url string) (*http.Response, error) {
```

Comment:
Should this be wrapping `util.MakeRequest`?

Reply:
Yeah and `util.MakeRequest` should set the `User-Agent` too instead of specifying it multiple times. Done